### PR TITLE
Withdrawn

### PR DESCRIPTION
**Pull request by dougdroper**
Frontend will be able to filter out withdrawn links based off this
withdrawn flag.

https://trello.com/c/Tv6uNqYq/857-expose-withdrawn-state-for-links

### CODE REVIEW

File: formats/frontend_links_definition.json [hunk 1]
```diff
@@ -62,6 +62,9 @@
       "analytics_identifier": {
         "$ref": "#/definitions/analytics_identifier"
       },
+      "withdrawn": {
+        "type": "boolean"
```
Comment (tijmenb) [inline formats/frontend_links_definition.json]: Can you add a description to this? Perhaps with a link to the publishing-api docs about what it means?

**Comment by tijmenb** [comment]
Currently the frontend links are a subset of the attributes in the frontend schema. Should we expose this element in the base content item too for consistency?

**Comment by fofr** [comment]
What's the user need for including withdrawn links?

**Comment by gpeng** [comment]
@fofr Publications that are `withdrawn` still require their attachments (which are also `withdrawn` in the Publishing API) to be rendered on the page otherwise they may no longer make sense to users as the 'actual' content is often in the attachment. Document Collections don't currently render links to `withdrawn` content so we need to be able to identify (and ignore) them for that format.

**Comment by dougdroper** [comment]
Spoke to @tijmenb about this and decided against exposing this element in the base content item.

**Pull request merged**